frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -886,6 +889,11 @@ const ManageHostsPage = ({
       // tableQueryData)
       handleTeamChange(teamId);
       handleResetPageIndex();
+      // Must clear other page paths or the team might accidentally switch
+      // When navigating from host details
+      setFilteredSoftwarePath("");
+      setFilteredQueriesPath("");
+      setFilteredPoliciesPath("");
```

Comment:
Switching to a team, going to run a live query of an existing query, click Hosts, change team, click on a host, click query, click on an existing query, click Back to queries, FIX still preserves team (bug was returning to previous team seen on queries page even though we changed the team when we went back to Hosts page)